packages/backend/src/scheduler/SchedulerTask.ts

```diff
@@ -314,14 +314,9 @@ export const sendSlackNotification = async (
             description: details.description,
             message: scheduler.message && slackifyMarkdown(scheduler.message),
             ctaUrl: url,
-            footerMarkdown:
-                schedulerUuid !== undefined
-                    ? `This is a <${url}?scheduler_uuid=${schedulerUuid}|scheduled delivery> ${getHumanReadableCronExpression(
-                          cron,
-                      )} from Lightdash\n${
-                          s3Client.getExpirationWarning()?.slack || ''
-                      }`
-                    : `This is a scheduled delivery triggered manually from Lightdash`,
```

Comment:
I removed this message entirely, I don't think it provides any value, and you probably want to see the cron footer on `send now`